src/main/java/games/strategy/triplea/delegate/UnitComparator.java

```diff
@@ -119,44 +61,11 @@
       final PlayerID player, final boolean noTies) {
     final Comparator<Unit> decreasingCapacityComparator = getDecreasingCapacityComparator(transports);
     final Predicate<Unit> incapableTransportMatch = Matches.transportCannotUnload(route.getEnd());
-    return (t1, t2) -> {
-
-      // Check if transport is incapable due to game state
-      final boolean isIncapable1 = incapableTransportMatch.test(t1);
-      final boolean isIncapable2 = incapableTransportMatch.test(t2);
-      if (!isIncapable1 && isIncapable2) {
-        return -1;
-      }
-      if (isIncapable1 && !isIncapable2) {
-        return 1;
-      }
-
-      // Prioritize allied transports
-      final boolean isAlliedTrn1 = !t1.getOwner().equals(player);
-      final boolean isAlliedTrn2 = !t2.getOwner().equals(player);
-      if (isAlliedTrn1 && !isAlliedTrn2) {
-        return -1;
-      }
-      if (!isAlliedTrn1 && isAlliedTrn2) {
-        return 1;
-      }
```

Comment:
Note that the expressions here are the opposite of the similar block above in `getLoadableTransportsComparator()`, but the return values are the same.  That is, this looks like it's a descending comparison.  You might have to remove the `reversed()` call below.